pkgs/development/python-modules/proton-core/default.nix

```
@@ -33,14 +35,13 @@ buildPythonPackage rec {
     requests
   ];
 
-  postPatch = ''
-    substituteInPlace setup.cfg \
-      --replace "--cov=proton --cov-report html --cov-report term" ""
-  '';
-
   pythonImportsCheck = [ "proton" ];
 
-  nativeCheckInputs = [ pytestCheckHook ];
+  dependencies = [
```

This is why pytestCheckPhase is being run:

```suggestion
  nativeCheckInputs = [
```

---

🤦🏻‍♂️ Thanks 👍🏻